jenkinsci/contrast-continuous-application-security-plugin
# Contrast 21733

Added functionality allowing to add global vulnerability policy.
To test:
- Create a new teamserver profile.
- Uncheck the checkbox "Allow global Contrast Vulnerability Threshold Conditions to be overridden in a Job configuration".
- Click "Apply" to save the profile,
- Add a new global Vulnerability Threshold Condition for the created profile.
- Create a new Job with the Contrast post-build step and build it.
- Make sure that the job console logs contain the correct global threshold condition.

![global policy](https://user-images.githubusercontent.com/18661283/42402421-448fe130-8140-11e8-80b6-ea2342cd80f6.png)
![job policy use global conditions](https://user-images.githubusercontent.com/18661283/42402422-44b23096-8140-11e8-8bc1-8c2382ca4408.png)
![job policy](https://user-images.githubusercontent.com/18661283/42402423-44d7cbbc-8140-11e8-9d59-b82ba710582f.png)
